Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM python:3.6-alpine
+FROM python:3.6
 
 WORKDIR /src
 
-COPY send_to_datadog.py /src
+COPY Pipfile pipenv.txt send_to_datadog.py /src/
 
-RUN pip3 install datadog
+RUN pip install -r pipenv.txt && \
+  pipenv install --dev
```

Comment:
I believe this will install into a virtual environment, but you probably want to install it into the system, using `--system` and (optionally) `--deploy` so if Pipfile is changed without updating the lock file you get an error.

Reply:
The CMD should probably also be `pipenv run python send_to_datadog.py` though that might not be needed if you use `--system` above.